src/components/SubNavigation/SubNavigation.scss

```diff
@@ -0,0 +1,145 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.dc-subnavigation {
+    $horizontalPadding: 12px;
+    $verticalPadding: 4px;
+
+    position: absolute;
+    z-index: 119;
+
+    display: flex;
+
+    width: calc(100% - 2 * #{$horizontalPadding});
+    height: #{$subNavigationHeight} - #{$verticalPadding} - #{$verticalPadding};
+    padding: #{$verticalPadding} #{$horizontalPadding};
+
+    gap: #{$verticalPadding};
+
+    background: var(--g-color-base-background, #fff);
+    border-bottom: 1px solid var(--g-color-line-generic, #0000001a);
+
+    transition: margin-top 300ms 0s;
+
+    @media (min-width: map-get($screenBreakpoints, 'md')) {
+        display: none;
+    }
+
+    .dc-root_wide-format & {
+        padding-right: 8px;
+        width: calc(100% - #{$horizontalPadding} - 8px);
+    }
+
+    &_visible {
+        margin-top: 0;
+    }
+
+    &_invisible {
+        --dc-header-height: 64px;
+        margin-top: calc(0px - var(--dc-header-height, #{$headerHeight}));
+    }
+
+    &_hidden {
+        border: none;
+        background: none;
+    }
+
+    &__icon {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        padding: 12px;
+    }
+
+    &__mini-toc-button {
+        display: flex;
+        align-items: center;
+        justify-content: flex-start;
+        flex: 1 0 0;
+
+        width: 100%;
+        padding: 0;
+        padding-left: 4px;
+        gap: 4px;
+
+        border-radius: var(--g-border-radius-xs, 3px);
+        border: none;
+        border-left: 1px solid var(--g-color-line-generic, rgba(0, 0, 0, 0.1019607843));
```

Comment:
what is `rgba(0, 0, 0, 0.1019607843)` ?

Reply:
This is the exact value of the border color taken from Figma